olive/model/__init__.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         self.local_model_path = None
         self.composite_parent = None
         self.io_config = None
+        self.auto_tune_config = None
```

Comment:
>auto_tune_config [](http://example.com/codeflow?start=13&length=16)

@jambayk , should be better if we keep it globally like engine config instead of in model itself? 

"auto_tune": {
    "num_heads": 128,
    ""hidden_size: 64
}